crates/gitbutler-branch-actions/tests/fixtures/dependencies.sh

```diff
@@ -2,6 +2,60 @@
 set -eu -o pipefail
 CLI=${1:?The first argument is the GitButler CLI}
 
+# Function to add a custom header to a commit
+#
+# Example usage:
+# set_change_id adbb3234 "change-id-1" "my_stack"
+set_change_id() {
+    local commit_hash=$1
+    local change_id=$2
+    local branch_name=$3
+
+    local change_id_key="gitbutler-change-id"
+    local gitbutler_header_version_key="gitbutler-headers-version"
+    local gitbutler_header_version_value="2"
+
+    if [ -z "$commit_hash" ] || [ -z "$change_id" ] || [ -z "$branch_name" ]; then
+        echo "Usage: set_change_id <commit_hash> <change_id> <branch_name>"
+        return 1
+    fi
+
+    # Get tree, parent, author, and committer information
```

Comment:
I'm a bit allergic to documentation that says "the next line will do x", and when you read the next line "it does x". Perhaps some of these strings are superfluous?